### ISSUE TITLE
Use of numpad can trigger collapse

### ISSUE THREAD

**Issue by chrona** [adobe/brackets#11054]
[ Windows 7 | Release 1.3 Build 1.30-16022 ]

1. Open any file which can collapse
2. Try to create a character using the numpad (e.g. an ellipsis -> ALT + 0133); a character using 1 is required.

=====

Result:
Everything is collapsed now.

Expected:
The ellipsis should appear without everything collapsing.

Workaround:
Using the character escaped or use copy/paste from somewhere else.

**Comment by ryanstewart** [comment]
This is because the key binding for code folding collapse all is alt+1. You can override this by modifying the keymap file - https://github.com/adobe/brackets/wiki/User-Key-Bindings

Also wouldn't it be better to use `&hellip;` instead of the Alt+ way of inputing it?

**Comment by chrona** [comment]
Yeah, I know why it collapses, I'd just assume that it wouldn't happen while using the numpad.
But the Key-Bindings seems fine for a workaround, thanks!

**Comment by js0511** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

oiup2006@naver.com 님께 보내신 메일 <Re: [brackets] Use of numpad can trigger collapse (#11054)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Comment by nethip** [comment]
@Naver As I mentioned earlier we keep this forum dedicated to issues and development work of Brackets and let us restrict it to these items alone. Please do not post about non relevant topics here.

**Comment by abose** [comment]
@nethip what is the comment about @Naver posting things? a google search says that naver is a billion$ corporation.

**Comment by nethip** [comment]
It is just spam. I wrote to GitHub support about this and they are looking into to. Basically that person has set a rule in his Inbox to send a reply to every notification from Brackets. :wink:

**Comment by abose** [comment]
If this is their valid Github handle, why do a corporation of this size want to spam us :open_mouth:

**Comment by thany** [comment]
I think one solution to this problem might be that Alt+number hotkeys should only work with the regular row of number keys, not with the numpad. This is because, as demonstrated, Alt+numpadX has special meaning in Windows.
